solutions/cf-782b.mdx

```diff
@@ -0,0 +1,104 @@
+---
+id: cf-782B
+source: CF
+title: The Meeting Place Cannot Be Changed
+author: Aditya Gupta
+---
+
+[Official Analysis](https://codeforces.com/blog/entry/50854)
+
+## Explanation
+
+We can binary search on the location where the friends will meet up. Since we want a precision of $10^{-6}$, we stop when the difference between high and low is less than $10^{-6}$. 
+
+Whenever we are calculating the answer, we take note of which side the maximum answer came from. If the max answer comes from both sides, regardless of where we, decide to go next, our answer will only increase, so we return that answer. Else, if it comes from only one side of the point that we just calculated, we move our search towards that side, as moving it towards the other side will only increase our answer. 
```

Comment:
```suggestion
Whenever we are calculating the answer, we take note of which side the maximum answer came from. If the max answer comes from both sides, regardless of where we decide to go next, our answer will only increase, so we return that answer. Otherwise, if it only comes from one side of the point that we just calculated, we move our search towards that side, as moving it towards the other side will only increase our answer. 
```